src/sentry/api/endpoints/release_thresholds/health_checks/is_new_issues_count_healthy.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import annotations
+
+from logging import getLogger
+
+from sentry.api.endpoints.release_thresholds.types import EnrichedThreshold
+from sentry.models.group import Group
+from sentry.models.groupenvironment import GroupEnvironment
+from sentry.models.release_threshold.constants import TriggerType
+
+logger = getLogger("sentry.releases.servicer")
+
+
+def is_new_issues_count_healthy(release_threshold: EnrichedThreshold) -> bool:
```

Comment:
‼️ 

So - looks like we aren't actually making any explicit requests to Snuba here??

What this looks like to me is that we actually _can_ make a single request for all of our new issues - and parse the results in python world 😄 

I'll leave a comment on the API file but the TLDR is that we can have a helper fetch function to query for all of our new issues, then in the api as we're iterating through the thresholds, we can parse the bulk data and pull the new issue counts per threshold.

This allows us to consolidate our queries to a single request and do the logic all on the python side